ckanext/canada/helpers.py

```diff
@@ -186,7 +186,7 @@ def adobe_analytics_lang():
         return 'fra'
 
 def adobe_analytics_js():
-    if config.get('ckan.site_url', '') == 'https://staging.open.canada.ca':
+    if config.get('ckanext.site_environment', '') == 'staging':
```

Comment:
No, what I mean is put the whole "//assets.adobetm.com..." string in a configuration variable instead of hard-coding the two values we're using behind an opaque "site_environment" setting

Reply:
Oh okay i see. Yes that would make more sense.